.github/workflows/build.yml

```
@@ -58,54 +60,61 @@ jobs:
     - run: ./res/ci/nixish_setup.sh && make bsd
     - env:
         MKN_LIB_LINK_LIB: 1
+        ASAN_OPTIONS: detect_leaks=0
       run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:3:7: Double quote to prevent globbing and word splitting

(shellcheck)